src/util/bluebird.js

```diff
@@ -1,7 +1,3 @@
-'use strict'
-
 // Use this shim module rather than "bluebird/js/main/promise"
 // when bundling for client
-module.exports = function() {
-  return require('bluebird')
-}
\ No newline at end of file
+export default () => require('bluebird')
```

Comment:
This is always inconsistent between files due to different editors. I'm not sure what the preference is here.